test/TestUtil.js

```diff
@@ -30,6 +30,37 @@ exports.promiseLater = function (time, func) {
     }));
 };
 
+exports.getConnections = function(client) {
+    if (Object.prototype.hasOwnProperty.call(client, 'connectionRegistry')) {
+        return client.connectionRegistry.getConnections();
+    } else {
+        return client.getConnectionManager().getActiveConnections();
```

Comment:
Minor: We don't need these compatibility codes in maintenance branches.

Reply:
ok I can remove them.